### ISSUE TITLE
Module Request: Spotify control

### ISSUE THREAD

**Issue by Deanbau** [bitfocus/companion-module-requests#57]
**Describe the feature**
Control spotify on a local or remote machine
**Is this platform dependent (windows, mac, ..)?**
https://github.com/dronir/SpotifyControl works for mac only

**If documentation is required to implement, do you know where to find it?**
See above project

**Usecases**
Control spotify, usually for BGM purposes.

**Comment by MarauderSeeker** [comment]
has any progress been made on this? I would really like this feature.

**Comment by josephdadams** [comment]
I don't think so. I currently do it via AppleScript since I am running Spotify on MacOS.

**Comment by Simber1** [comment]
> has any progress been made on this? I would really like this feature.
@MarauderSeeker 
Might be 6 months late but if you're still interested I'm working on this now and it's coming along quite nicely.

**Comment by MarauderSeeker** [comment]
Great

**Comment by daniellippens** [comment]
> > has any progress been made on this? I would really like this feature.
> 
> @MarauderSeeker
> Might be 6 months late but if you're still interested I'm working on this now and it's coming along quite nicely.

Very nice! I'm interested in controlling spotify over network/the web as opposed to locally.

**Comment by josephdadams** [comment]
Transferring this to the Spotify module repository and you guys can close it out when you're ready.
